snappy/overlord.go

```diff
@@ -342,6 +344,49 @@ func (o *Overlord) SetActive(s *SnapPart, active bool, meter progress.Meter) err
 	return s.deactivate(false, meter)
 }
 
+// ConfigureFromSnippet turns snippet into a snap configuration
+// and calls Configure on the result.
```

Comment:
Please explain what the snippet is. I had to run back and forth in this diff to catch that.